src/main/kotlin/com/grepp/nbe1_3_team04/chat/api/request/ChatMemberRequest.kt

```diff
@@ -0,0 +1,14 @@
+package com.grepp.nbe1_3_team04.chat.api.request
+
+import com.grepp.nbe1_3_team04.chat.service.request.ChatMemberServiceRequest
+import jakarta.validation.constraints.NotNull
+
+@JvmRecord
+data class ChatMemberRequest(
+    val chatroomId: @NotNull(message = "채팅방 아이디는 필수입니다.") Long?,
```

Comment:
```suggestion
    val chatroomId: @field:NotNull(message = "채팅방 아이디는 필수입니다.") Long?,
```

이 부분 field 추가 안해도 NotNull 체크 제대로 해주나요??

Reply:
이 부분에 대해서 찾아봤는데 제가 작성한 코드는 타입에 애노테이션을 적용시킨거고 이런 형식은 자바 스타일이라 코틀린과 호환이 안좋다고 하네요
재경님이 말씀주신 것처럼 코틀린에서는 필드에 어노테이션 선언하는데 권장된다고 합니다.

```suggestion
    @field:NotNull(message = "채팅방 아이디는 필수입니다.")
    val chatroomId: Long?,
```
아예 이렇게 필드에 적용했다고 작성하는게 직관적이고 좋아 보입니다.

수정하겠습니다!